Fix:
./arp.c:76:4: error: implicitly declaring library function 'memcpy' with type 'void *(void *, const void *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
--- arp.c.orig	2001-03-15 02:32:58.000000000 -0600
+++ arp.c	2021-07-29 10:51:57.000000000 -0500
@@ -33,6 +33,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <unistd.h>
 
 #include "arp.h"
